liberty-maven-plugin/src/it/springboot-3-appsdirectory-apps-fail-it/pom.xml

```diff
@@ -0,0 +1,170 @@
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>springboot-3-appsdirectory-apps-fail-it</artifactId>
+	<version>1.0.0.Final</version>
+	<packaging>jar</packaging>
+
+	<parent>
+		<groupId>io.openliberty.tools.it</groupId>
+		<artifactId>tests</artifactId>
+		<version>1.0-SNAPSHOT</version>
+	</parent>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<maven.compiler.source>17</maven.compiler.source>
+		<maven.compiler.target>17</maven.compiler.target>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+			<version>3.1.3</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<version>3.1.3</version>
+			<scope>test</scope>
+		</dependency>
+                <dependency>
+                    <groupId>junit</groupId>
+                    <artifactId>junit</artifactId>
+                    <version>4.13.2</version>
+                    <scope>test</scope>
+                </dependency>	
+ 		<!-- Support for JDK 9 and above-->
+    		<dependency>
+        		<groupId>javax.xml.bind</groupId>
+        		<artifactId>jaxb-api</artifactId>
+        		<version>2.3.1</version>
+        		<scope>provided</scope>
+    		</dependency>        
+		<dependency>
+        		<groupId>com.sun.xml.bind</groupId>
+        		<artifactId>jaxb-core</artifactId>
+        		<version>2.3.0.1</version>
+        		<scope>provided</scope>
+    		</dependency>
+    		<dependency>
+        		<groupId>javax.activation</groupId>
+        		<artifactId>activation</artifactId>
+        		<version>1.1.1</version>
+        		<scope>provided</scope>
+    		</dependency>	
+	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-maven-plugin</artifactId>
+				<version>3.1.3</version>
+				<!-- Test support for classifier configuration parameter -->
+				<configuration>
+					<classifier>exec</classifier>
+				</configuration>
+				<executions>
+					<execution>
+						<goals>
+							<goal>repackage</goal>
+						</goals>
+						<configuration>
+							<skip>true</skip>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+
+			<plugin>
+				<groupId>io.openliberty.tools</groupId>
+				<artifactId>liberty-maven-plugin</artifactId>
+				<version>@pom.version@</version>
+
+				<!-- Specify configuration, executions for liberty-maven-plugin -->
+				<configuration>
+					<serverName>test</serverName>
+                    <serverXmlFile>src/test/resources/server.xml</serverXmlFile>
```

Comment:
This is not the right server.xml that you provided for the test with the duplicate springBootApplication element.

Reply:
pom corrected